base/multidimensional.jl

```diff
@@ -1755,14 +1755,18 @@ julia> unique(A, 2)
   true
  false
 
-julia> unique(A, 3)
+julia> unique(A, dims=3)
 2×2×1 Array{Bool,3}:
 [:, :, 1] =
   true   true
  false  false
 ```
 """
-@generated function unique(A::AbstractArray{T,N}, dim::Int) where {T,N}
+@generated function unique(A::AbstractArray{T,N}; dims::Union{Colon,Integer} = :) where {T,N}
+    if dims === (:)
+        return invoke(unique, Tuple{Any}, A)
+    end
+    dim = dims
```

Comment:
I don't think this will work — you're in a `@generated` function here.

Reply:
Oops, thanks.